src/Factory/Psr17Factory.php

```diff
@@ -52,7 +52,7 @@ public function createStreamFromResource($resource): StreamInterface
         return Stream::create($resource);
     }
 
-    public function createUploadedFile(StreamInterface $stream, int $size = null, int $error = \UPLOAD_ERR_OK, string $clientFilename = null, string $clientMediaType = null): UploadedFileInterface
+    public function createUploadedFile(StreamInterface $stream, int $size = null, int $error = UPLOAD_ERR_OK, string $clientFilename = null, string $clientMediaType = null): UploadedFileInterface
```

Comment:
maybe these cs changes are not needed?

Reply:
Very true.. php-cs-fixer suggested it, I just blindly approved.

Reply:
Rules updated!